### PR TITLE
Feature/best practices metadata section

### PR DESCRIPTION
**Pull request by avani-k**

### CODE REVIEW

File: docs/advanced-topics/best-practices/best-practices-secure-fair-workflows.rst [hunk 1]
```diff
@@ -105,9 +105,14 @@ Findable
 		
 	- Authorship, contact information, and description:
 	
-		- You can add author and description metadata to your descriptor file. Adding an author will make it selectable on the Author facet in Dockstore’s search and a description helps because the text search uses it as one of the fields to sift through. 
+		- You can add author and description metadata to your descriptor file. Adding an author will make it selectable on the Author facet in Dockstore’s search and a description helps because the text search uses it as one of the fields to sift through.
+		
+	- Link GitHub repository:
+	
+		- Additionally, you can include a link to your original GitHub repo in order to connect the Dockstore entry with the source code. If your workflow is moved out of Dockstore, this can help to keep it connected with the original source code. You may consider doing so especially when you have additional comprehensive README files available on GitHub.
```
Comment (bethsheets) [inline docs/advanced-topics/best-practices/best-practices-secure-fair-workflows.rst:112]: "Link github repo" is a bit confusing here. I think we are suggested they paste their GitHub repo readme URL in the description metadata section of the WDL, correct? I may clarify the language for that.

File: docs/advanced-topics/best-practices/best-practices-secure-fair-workflows.rst [hunk 2]
```diff
@@ -251,4 +256,6 @@ Do not use untagged or “latest”.
 			- Any available cost or benchmarking information. 
 			- How to cite the use of your workflow (including references for the original software authors). 
 
+- Note: Documentation can be housed at either or both, the metadata section of the workflow file and/or the GitHub README document. On Dockstore, if a description is provided in the metadata section, it will be displayed by default in the INFO section. If a metadata section is missing, Dockstore will pull information from the README file on the original GitHub repo.
```
Comment (bethsheets) [inline docs/advanced-topics/best-practices/best-practices-secure-fair-workflows.rst:259]: This looks good!

**Review by bethsheets — CHANGES_REQUESTED** [review]
One suggestion for clarification in the instructions; otherwise, looks nice!

**Comment by avani-k** [comment]
Requested changes made!

**Pull request closed without merging**